py/selenium/webdriver/remote/webdriver.py

```diff
@@ -947,15 +947,22 @@ def add_cookie(self, cookie_dict):
 
         :Args:
          - cookie_dict: A dictionary object, with required keys - "name" and "value";
-            optional keys - "path", "domain", "secure", "expiry"
+            optional keys - "path", "domain", "secure", "expiry", "sameSite"
 
         Usage:
             driver.add_cookie({'name' : 'foo', 'value' : 'bar'})
             driver.add_cookie({'name' : 'foo', 'value' : 'bar', 'path' : '/'})
             driver.add_cookie({'name' : 'foo', 'value' : 'bar', 'path' : '/', 'secure':True})
+            driver.add_cookie({'name': 'foo', 'value': 'bar', 'sameSite': 'Strict'})
 
         """
-        self.execute(Command.ADD_COOKIE, {'cookie': cookie_dict})
+        if 'sameSite' in cookie_dict:
+          if cookie_dict['sameSite'] =='Strict' or cookie_dict['sameSite'] == 'Lax':
```

Comment:
This would be better as 
```suggestion
          if cookie_dict['sameSite'] =='Strict' or cookie_dict['sameSite'] == 'Lax':
          if cookie_dict['sameSite'] in ['Strict', 'Lax']:
```

Reply:
Updated as suggested!